core/ui/prime/components/src/lib/input-textarea/input-textarea.component.ts

```diff
@@ -12,4 +12,13 @@ import { InputBaseComponent } from "../input-base/input-base.component";
 export class InputTextareaComponent extends InputBaseComponent {
     @Input() id!: string;
     @Input() placeholder?: string;
+    @Input() autoResize: boolean;
```

Comment:
`@Input() autoResize = false;`
Is the same as initializing `autoResize` in the constructor.

Reply:
Code updated